app/src/test/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreenTest.kt

```diff
@@ -0,0 +1,283 @@
+package co.nimblehq.compose.crypto.ui.screens.home
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.*
+import androidx.navigation.*
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.domain.usecase.GetMyCoinsUseCase
+import co.nimblehq.compose.crypto.domain.usecase.GetTrendingCoinsUseCase
+import co.nimblehq.compose.crypto.extension.toFormattedString
+import co.nimblehq.compose.crypto.test.MockUtil
+import co.nimblehq.compose.crypto.ui.navigation.AppDestination
+import co.nimblehq.compose.crypto.ui.screens.BaseViewModelTest
+import co.nimblehq.compose.crypto.ui.screens.MainActivity
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.*
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.shadows.ShadowToast
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
@Wadeewee Can you make this UI test run with the Instrument test first?

Reply:
Yup, @luongvo, but let me create a new issue for UI Test and open it in a new PR soon 🙏

Reply:
@Wadeewee Should we go with the correct steps: make the UI test work with the Instrument test first, and then clone or move it to unit test with Robolectric? cc @hoangnguyen92dn

Reply:
@luongvo May I know why you want to see the `UI Test` first 🙏

Reply:
FYI: @luongvo @hoangnguyen92dn, Here's for  [HomeScreenUITest](https://github.com/nimblehq/jetpack-compose-crypto/pull/86) 🚀